integral_view/views/ntp_management.py

```diff
@@ -98,4 +101,24 @@ def update_ntp_settings(request):
         return django.shortcuts.render_to_response("logged_in_error.html", return_dict, context_instance=django.template.context.RequestContext(request))
 
 
+def sync_ntp(request):
+    return_dict = {}
+    try:
+        output, err = ntp.sync_ntp()
+        if err:
+            raise Exception(err)
+        else:
+            if 'ntp_sync' in output and output['ntp_sync'] == True and 'server_used' in output:
+                return django.http.HttpResponseRedirect("/view_ntp_settings?ack=ntp_synced&server_used=%s" % output['server_used'])
+            else:
+                raise Exception("Couldn't sync because the required parameters doesn't met")
```

Comment:
The comment will not help the user debug what the problem is. So we should either say that the sync failed or say that there were no servers configured. When does the function call fail? We should somehow inform the user why it failed so they can fix the issue somehow.